task1/lib/main.dart

```diff
@@ -0,0 +1,157 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Task 1',
+      home: MainPage(),
+    );
+  }
+}
+
+class MainPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(centerTitle: true, title: Text('Flutter Vadodara')),
+      body: Container(
+        padding: EdgeInsets.only(top: 16, bottom: 16),
+        margin: EdgeInsets.fromLTRB(32, 0, 32, 0),
+        child: Column(
+          children: [
+            Container(
+                color: Colors.blue,
+                width: 250.0,
+                height: 100,
+                margin: EdgeInsets.all(30),
+                child: Center(
+                  child: Text("Flirt",
+                      textAlign: TextAlign.center,
+                      style: TextStyle(
+                          fontStyle: FontStyle.italic, fontSize: 60.0)),
+                )),
+            Container(
+                color: Colors.orange,
+                width: 250.0,
+                height: 100,
+                margin: EdgeInsets.all(30),
+                child: Center(
+                  child: Text("With",
+                      textAlign: TextAlign.center,
+                      style: TextStyle(
+                          fontStyle: FontStyle.italic, fontSize: 60.0)),
+                )),
+            Container(
+                color: Colors.tealAccent,
+                width: 250.0,
+                height: 100,
+                margin: EdgeInsets.all(30),
+                child: Center(
+                  child: Text("Flutter",
+                      textAlign: TextAlign.center,
+                      style: TextStyle(
+                          fontStyle: FontStyle.italic, fontSize: 60.0)),
+                )),
+          ],
+          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+        ),
+      ),
+      floatingActionButton: FloatingActionButton(
+        onPressed: () {
+          navigateToSubPage(context);
+        },
+        child: Icon(Icons.near_me),
+      ),
+    );
+  }
+}
+
+Future navigateToSubPage(context) async {
+  Navigator.push(context, MaterialPageRoute(builder: (context) => SubPage()));
+}
+
+class SubPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Back to Main Page'),
+        backgroundColor: Colors.blue,
+      ),
+      body: Column(
```

Comment:
If you read the docs, I have mentioned having margin of 32 from left and right and padding of 16 from top and bottom.